docs/tutorials/portable-consent/subscribe.md

```diff
@@ -0,0 +1,221 @@
+---
+sidebar_label: Subscribe
+sidebar_position: 4
+---
+
+# Enable subscribers with XMTP
+
+![Feature status](https://img.shields.io/badge/Feature_status-Alpha-orange)
+
+:::caution This feature is in **alpha** status
+
+This feature is in **alpha** status and ready for you to start experimenting with. However, we do not recommend using alpha features in production apps. Expect frequent changes as we iterate based on feedback. Want to provide feedback? Comment on [Proposal: Portable consent state for v2 SDKs](https://github.com/orgs/xmtp/discussions/49).
+
+:::
+
+<div class=" rabbit  p-5 ">
+
+📥 <b>Need a quick reference?</b> Check out this GitHub repo: <a href="https://github.com/fabriguespe/xmtp-subscribe-portable-consent">xmtp-subscribe</a>
+
+</div>
+
+This tutorial will show you how to create a simple **Subscribe** button with consent that enables users to subscribe to your messages or notifications.
+
+## Import libraries
+
+Import the necessary XMTP and Ethereum packages. These libraries enable you to create an XMTP client and interact with the Ethereum blockchain.
+
+```jsx
+import React, { useEffect, useState } from "react";
+import { Client } from "@xmtp/xmtp-js";
+import { ethers } from "ethers";
+```
+
+The receiver of the subscription is you. In this case, we'll make it a random wallet generated by `ethers`, but you could send your own wallet address as a prop. This wallet is later going to
```

Comment:
@fabriguespe I think there are a few words missing at the end of this paragraph? TY!